### DISCUSSION TITLE
Where is the feature list for this mod?  Does it apply to servers?

### DISCUSSION THREAD

**Discussion by allanonmage** [natanfudge/Not-Enough-Crashes#166] [General]
The title of the mod implies that it causes more crashes, which is probably tongue-in-cheek.  But the description only lists 1 or 2 things that the mod does, not much.  If there wasn't as much activity on the Github, I'd just pass it on.  What is this mod for and why should I use it?

I was banned from Discord as i was signing up to it, so it's not like I can go check out your Discord server.  Since I've never used Discord, I don't even know what all the pizzazz is about so I am completely unmotivated to try harder to join it.

**Comment by natanfudge** [comment]
The feature list on the description is pretty exhaustive. Not closing the game on a crash can save like 20 minutes each time it happens, which is pretty useful. On the server, it will display a list of 'suspected mods' of the crash in crash logs.
